adhocracy4/rules/templatetags/discovery_tags.py

```diff
@@ -0,0 +1,14 @@
+from django import template
+
+from adhocracy4.rules.discovery import NormalUser
+
+register = template.Library()
+
+
+@register.assignment_tag
```

Comment:
The assignment tag is deprecated since django 1.9. We should use simple tags instead.

Reply:
To keep it compatible with 1.8 and OPIN, we need to use the assignment_tag.